test/io/serializers/warn-interfaces.good

```diff
@@ -0,0 +1,48 @@
+warn-interfaces.chpl:nnnn: warning: the type 'JustTwoRecord' defines methods that previously had special meaning. These will soon require 'JustTwoRecord' to implement the 'initDeserializable' interface to continue to be treated specially.
```

Comment:
This is good IMO; however, I think the phrase "previously had special meaning" has a high likelihood of being pretty confusing, particularly for people who aren't familiar with the phrase "special-method".

I might adjust to something like:

`the type 'JustTwoRecord' defines methods belonging to an interface; however, it does not explicitly implement the interface which will be required in a future release. Please add ': initDeserializable' to the type declaration to continue using the methods as a part of the interface.`

Reply:
Offline we concluded to save this discussion for another time, since the warnings were pre-existing and apply to other cases as well (e.g. hashable).